ops/platform-dev/base/deployment.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: content-library
+  labels:
+    app: content-library
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: content-library
+  template:
+    metadata:
+      labels:
+        app: content-library
+    spec:
+      serviceAccountName: content-library
+      containers:
+      - name: content-library
+        image: gcr.io/adhoc-dev-service-platform/content-library:latest
+        ports:
+          - containerPort: 3000
+        livenessProbe:
+          httpGet:
```

Comment:
note that if the liveness probe failed it would show up like this
```
  Warning  Unhealthy  2m46s (x9 over 4m16s)  kubelet            Liveness probe failed: Get "http://10.17.2.7:2322/healthcheck": dial tcp 10.17.2.7:2322: connect: connection refused
  ```